core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -170,6 +183,137 @@ export class ContributionAndReviewService {
     );
   }
 
+  async fetchTranslationSuggestionsAsync(
+      explorationId: string
+  ): Promise<FetchSuggestionsResponse> {
+    const explorationBackendResponse = await this.
+      readOnlyExplorationBackendApiService.fetchExplorationAsync(
+        explorationId, null);
+    return (
+      this.contributionAndReviewBackendApiService.
+        fetchSuggestionsAsync(
+          'REVIEWABLE_TRANSLATION_SUGGESTIONS',
+          null,
+          0,
+          AppConstants.SUGGESTIONS_SORT_KEY_DATE,
+          explorationId
+        ).then((fetchSuggestionsResponse) => {
+          const exploration: Exploration = this.explorationObjectFactory.
+            createFromExplorationBackendResponse(
+              explorationBackendResponse);
+          const sortedTranslationSuggestions = (
+            this.sortTranslationSuggestionsByState(
+              fetchSuggestionsResponse.suggestions,
+              exploration.getStates(),
+              exploration.initStateName));
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          const responseSuggestionIdToDetails: {[key: string]: any} = {};
```

Comment:
Here too. We should have a type for this.

Reply:
Done